2-mandatory/1-create-functions.js

```diff
@@ -33,20 +39,42 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(arr, i) {
+  return arr.filter((el) => el !== arr[i]);
```

Comment:
How about if the value of `arr[i]` occurs more than once in `arr`? eg try an input of `[1, 2, 1, 3, 4]`